src/components/ButtonPanel.js

```diff
@@ -1,16 +1,24 @@
 import React from 'react';
 import Button from './Button';
 
-const ButtonPanel = () => (
-  <div className="main">
-    <>
-      <Button buttons={['AC', '+/-', '%', '/']} />
-      <Button buttons={['7', '8', '9', 'X']} />
-      <Button buttons={['4', '5', '6', '-']} />
-      <Button buttons={['1', '2', '3', '+']} />
-      <Button buttons={['0', '.', '=']} />
-    </>
-  </div>
-);
+const ButtonPanel = () => {
+  const group1 = ['AC', '+/-', '%', '/'];
+  const group2 = ['7', '8', '9', 'X'];
+  const group3 = ['4', '5', '6', '-'];
+  const group4 = ['1', '2', '3', '+'];
+  const group5 = ['0', '.', '='];
+
+  return (
+    <div className="main">
+      <>
+        <Button name={group1} />
+        <Button name={group2} />
+        <Button name={group3} />
+        <Button name={group4} />
+        <Button name={group5} />
+      </>
+    </div>
+  );
```

Comment:
- [ ] why are you using the [`React.fragment`](https://reactjs.org/docs/fragments.html) if you're wrapping the component in a div? `React.fragment` is supposed to make your components return elements with nothing wrapping them, so wrapping it in a div destroys its purpose, either use the div or the fragment.

- [ ] Use divs here to create multiple rows of `Button` components, example: 

```jsx
<div>
    <Button name"AC" >
    <Button name"+/-" >
    <Button name"%" >
    <Button name"/" >
  </div>
  <div>
    <Button name"7" >
    <Button name"8" >
    <Button name"9" >
    <Button name"X" >
  </div>
  <div>
    <Button name"4" >
    <Button name"5" >
    <Button name"6" >
    <Button name"-" >
  </div>
  <div>
    <Button name"1" >
    <Button name"2" >
    <Button name"3" >
    <Button name"+" >
  </div>
  <div>
    <Button name"0" >
    <Button name"." >
    <Button name"=" >
  </div>
```